### PR TITLE
Use cluster-based inbox

### PR DESCRIPTION
**Pull request by paulgb**
This will enable a drone to operate even if it is restricted to messaging on `cluster.{cluster}.>`

Also bumps version.

### CODE REVIEW

File: controller/src/plan.rs [hunk 1]
```diff
@@ -21,7 +21,7 @@ pub struct ControllerPlan {
 
 impl ControllerPlan {
     pub async fn from_controller_config(config: ControllerConfig) -> Result<Self> {
-        let nats = config.nats.connect_with_retry().await?;
+        let nats = config.nats.connect_with_retry("controller.inbox").await?;
```
Comment (pretentious7) [inline controller/src/plan.rs:24]: so If I understand correctly, responses will be published to "controller.inbox.<UID>" ?

**Review by pretentious7 — APPROVED** [review]
one clarifying question, LGTM!

File: controller/src/plan.rs (hunk 1 above)
Comment (paulgb) [reply controller/src/plan.rs:24]: It will be `controller.inbox.{some random string generated by NATS}`

**Review by rolyatmax — APPROVED** [review]
LGTM! Nice that inbox-prefixing was already supported by NATS!

**Pull request merged**